migrations/20170419122022_make-is_primary-not-nullable.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+exports.up = (knex) => (
+  knex.schema
+    .raw('UPDATE records SET is_primary = false where is_primary IS NULL')
+    .raw('ALTER TABLE records ALTER COLUMN is_primary SET NOT NULL')
+);
+
+exports.down = (knex) => (
+  knex.schema
+    .raw('ALTER TABLE records ALTER COLUMN is_primary DROP NOT NULL')
```

Comment:
Although I understand why this can be useful, I don't think it's a good idea. The invariant of a down migration is that it must leave the database in the same state as it was before. We can't do it in this case, so I'd prefer to raise an Exception.